package.json

```diff
@@ -94,11 +94,13 @@
     "boom": "^5.2.0",
     "cids": "^0.5.1",
     "debug": "^3.0.0",
+    "fnv1a": "^1.0.1",
     "fsm-event": "^2.1.0",
     "glob": "^7.1.2",
     "hapi": "^16.5.2",
     "hapi-set-header": "^1.0.2",
     "hoek": "^4.2.0",
+    "interface-datastore": "^0.3.0",
```

Comment:
To `get` or `put` in the datastore, the first argument should be a datastore Key object. Based on the example [here](https://github.com/ipfs/interface-datastore) it seems like I need to import this like `const Key = require('interface-datastore').Key`